source/agora/registry/main.d

```diff
@@ -65,6 +70,10 @@ private int main (string[] args)
         return -1;
     }
 
+    StatsServer stats_server;
+    if (cmdline_args.stats_port != 0)
```

Comment:
once I got scolded for using port 0 for disabling stats server
https://github.com/bosagora/agora/pull/1249#discussion_r502983154

Reply:
Well, I don't see why anyone would want to bind a server socket to a random port so I think we are fine:)

Reply:
> Well, I don't see why anyone would want to bind a server socket to a random port so I think we are fine:)

"I can't think of any way someone would use that" is a sure way to be surprise :)
I remember a system where that was done, as the number of nodes to spawn wasn't known (and varied during the system lifetime), and there was simply a way for consumers to discover ports. So it's not *so odd*.